pylintrc

```diff
@@ -1,5 +1,5 @@
 [MESSAGES CONTROL]
-disable=fixme,locally-disabled,locally-enabled,relative-import,too-few-public-methods,missing-docstring,ungrouped-imports,too-many-lines
+disable=fixme,locally-disabled,locally-enabled,relative-import,too-few-public-methods,missing-docstring,ungrouped-imports,too-many-lines,import-error
```

Comment:
duplicated as the inline ones? maybe don't do it globally

Reply:
yeah, there need to be more inline ones, I will move this to more inline and fix the other lint errors but I wanted to check the idea out first. this was just a faster way to make sure it worked.

Reply:
done.